scripts/minikube.sh

```diff
@@ -93,6 +93,7 @@ up)
     enable_psp
 
     echo "starting minikube with kubeadm bootstrapper"
+    minikube addons disable storage-provisioner || true
```

Comment:
looks like its failing https://travis-ci.org/github/ceph/ceph-csi/jobs/665699624#L379

Reply:
can you disable default-storageclass` and `storage-provisioner` and check?

Reply:
It seems that I had to disable the EXTRA_OPTIONS (PodSecurityPolicy) as well to make it work.

Reply:
In that case we need to open an issue in minikube to track it. Disabling psp is not a best option as we are good now with current version of minikube